components/Stats.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
+import { Row, Col, Container } from "react-bootstrap";
 
 const Stats = () => {
-	return <div>Stats</div>;
+	return (
+		<div>
+			<Container>
+				<Row className='font-lexend text-4xl decoration-black font-semibold'>
```

Comment:
To make it centered and evenly spaced out on smaller screens, you can add `flex justify-between items-center`